infrastructure/ansible/roles/admin_user/defaults/main.yml

```diff
@@ -0,0 +1 @@
+admin_user: triplea
```

Comment:
This is notable, the 'admin_user' is the system-admin user that will be used to log in to the servers. This was previously called 'admin'. 
Because the 'admin' group is special, creating a user 'admin' usually creates an 'admin' group, the potential for confusion/conflict is worth avoiding and hence the name change update. Furthermore, we used to run everything as 'triplea', now we'll have app-specific system users.